e2e/mock-api-v2/vite.config.ts

```diff
@@ -5,17 +5,14 @@ export default defineConfig({
   cacheDir: '../../node_modules/.vite/e2e/mock-api-v2',
 
   test: {
-    globals: true,
-    cache: {
-      dir: '../../node_modules/.vitest',
-    },
-    watch: !process.env['CI'],
+    globals: false,
+    watch: false,
     environment: 'jsdom',
+    pool: 'forks',
```

Comment:
tests were hanging, adding this made them not hang. idk why but there was a large gh issue on this that should be out dated but this fixed it for me.